src/strategies/uniswap-v3/examples.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "name": "Uniswap-v3 query",
+    "strategy": {
+      "name": "uniswap-v3",
+      "params": {
+        "poolAddress": "0xfc9f572124d8f469960b94537b493f2676776c03"
```

Comment:
Could you add a way to select the token that you are interested in. As far as i understand now it calculate only voting power using token0 balance, but some project may want to use token1 for example.

Reply:
@bonustrack Updated